plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemFireworkPower.java

```diff
@@ -0,0 +1,45 @@
+package com.denizenscript.denizen.objects.properties.item;
+
+import com.denizenscript.denizen.objects.ItemTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.inventory.meta.FireworkMeta;
+
+public class ItemFireworkPower extends ItemProperty<ElementTag> {
+
+    // <--[property]
+    // @object ItemTag
+    // @name firework_power
+    // @input ElementTag(number)
+    // @description
+    // Controls the firework's power.
+    // Power primarily affects how high the firework flies, with each level of power corresponding to approximately half a second of additional flight them.

```

Comment:
`of additional flight them.` typo, should be `time` (this was typo'd in the original meta, not your fault, but should be fixed)